chaoscenter/cypress/e2e/UserManagment.cy.js

```diff
@@ -0,0 +1,105 @@
+describe('testing UserManagement', () => {
+    before(() => {
+        cy.requestLogin(Cypress.env('username'),Cypress.env('password'));
+    });
+
+    it('tesing through REST APIs', () => {
+        const accessToken = localStorage.getItem('accessToken');
+
+        //add user
+        const add_payload = {
+            name: 'user10',
+            email: 'user10@gmail.com',
+            username: 'user10',
+            password: 'user10',
+            role: 'user'
+          };
```

Comment:
We can shift this to fixtures & use from there.

Example - https://github.com/litmuschaos/litmus-e2e/blob/master/Cypress/cypress/fixtures/Users.json
Usage - https://github.com/litmuschaos/litmus-e2e/blob/d8cbeaf8661de20da49b6fa62146eafa5202f489/Cypress/cypress/integration/Basic_Setup/1.%20Login/login.spec.js#L18